### PR TITLE
Add header text to first run setup screens

### PR DESCRIPTION
**Pull request by peppy**
- [x] Depends on framework bump (for https://github.com/ppy/osu-framework/pull/5134).

osu! 2022-04-27 at 09 48 23

### CODE REVIEW

**Review by frenzibyte — COMMENTED** [review]
Looks good. Waiting for framework bump before approval.

**Review by bdach — APPROVED** [review]

**Pull request merged**
